elephant/asset.py

```diff
@@ -612,101 +599,93 @@ def cluster_matrix_entries(mat, eps=10, min_neighbors=2, stretch=5):
 def probability_matrix_montecarlo(
         spiketrains, binsize, spiketrains_y=None,
         t_start_x=None, t_start_y=None, t_stop_x=None, t_stop_y=None,
-        surr_method='dither_spike_train', j=None, n_surr=100, verbose=False):
+        surr_method='dither_spike_train', jitter=None, n_surr=100,
```

Comment:
Parameter named `surrogate_method` is clearer.
As in the other functions, `binsize` should be `bin_size`.
And `n_surr` should be `n_surrogates`.

Reply:
I agree with all